docker/Dockerfile

```diff
@@ -1,22 +1,25 @@
 ARG ROS_DISTRO
 ARG UBUNTU_VERSION
 FROM osrf/ros:${ROS_DISTRO}-desktop-${UBUNTU_VERSION}
+ENV DEBIAN_FRONTEND noninteractive
 
+RUN apt-key adv --keyserver 'hkp://keyserver.ubuntu.com:80' --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
```

Comment:
Sorry, but what is this for?

Reply:
this is for `apt update`.
gpg key is wrong so that `apt update` fails and cannot install wget without doing it.

Reply:
I checked again, and it seems we do not need this line probably.
I will remove the line.

Reply:
Do you mean `osrf/ros:${ROS_DISTRO}-desktop-${UBUNTU_VERSION}` image is broken?
I think kinetic image and melodic image are not broken, because they seem to have new GPG key:
https://github.com/osrf/docker_images/blob/master/ros/kinetic/ubuntu/xenial/ros-core/Dockerfile#L12
https://github.com/osrf/docker_images/blob/master/ros/melodic/ubuntu/bionic/ros-core/Dockerfile#L19
However, I found indigo image may be broken, because it seems to have old GPG key:
https://github.com/osrf/docker_images/blob/master/ros/indigo/ubuntu/trusty/ros-core/Dockerfile#L12
Does your problem occur on indigo image?
If so, we should create an issue in https://github.com/osrf/docker_images and comment that issue URL to this line.

Reply:
i update and remove the line